Add support for modern keyrings (#1128)
puppetlabs/puppetlabs-apt: README.md

<a id="add-gpg-keys"></a>

### Add GPG keys
You can fetch GPG keys via HTTP, Puppet URI, or local filesystem. The key must be in binary format for apt to read it properly.

> #### Fetch via HTTP
> ```puppet
> apt::keyring {'puppetlabs-keyring.gpg':
>   source => 'https://apt.puppetlabs.com/keyring.gpg',
> }
> ```
> #### Fetch via Puppet URI
> ```puppet
> apt::keyring {'puppetlabs-keyring.gpg':
>   source => 'puppet:///modules/my_module/local_puppetlabs-keyring.gpg',
> }
> ```

Reviewer: ```suggestion
#### Fetch via HTTP

```puppet
apt::keyring {'puppetlabs-keyring.gpg':
  source => 'https://apt.puppetlabs.com/keyring.gpg',
}
```

#### Fetch via Puppet URI

```puppet
apt::keyring {'puppetlabs-keyring.gpg':
  source => 'puppet:///modules/my_module/local_puppetlabs-keyring.gpg',
}
```

```

markdown linter demands empty lines between blocks

<a id="add-gpg-keys"></a>

### Add GPG keys

> You can fetch GPG keys via HTTP, Puppet URI, or local filesystem. The key must be in binary format for apt to read it properly.

Reviewer 2: GPG binary and ASCII armored keys are supported in apt >= 1.4
```suggestion
You can fetch GPG keys via HTTP, Puppet URI, or local filesystem. The key can be in GPG binary format, or ASCII armored, but the filename should have the appropriate extension (`.gpg` or `.asc`)
```

### Add GPG keys

> You can fetch GPG keys via HTTP, Puppet URI, or local filesystem. The key can be in GPG binary format, or ASCII armored, but the filename should have the appropriate extension (`.gpg` or `.asc`).

Reviewer 3: TIL:

```suggestion
You can fetch GPG keys via HTTP, Puppet URI, or local filesystem. The key can be in GPG binary format, or ASCII armored, but the filename should have the appropriate extension (`.gpg` for keys in binary format; or `.asc` for ASCII armored keys).
```
